website/README.md

```
@@ -42,9 +41,9 @@ Follow these instructions to get the project running on your local machine.
     ```bash
     cd website
     ```
-3.  Install the dependencies using `pnpm`:
+3.  Install the dependencies using `npm`:
     ```bash
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)